sdk/textanalytics/azure-ai-textanalytics/README.md

```diff
@@ -35,7 +35,7 @@ Text Analytics supports both [multi-service and single-service access][service_a
 resource if you plan to access multiple cognitive services under a single endpoint/key. For Text Analytics access only,
 create a Text Analytics resource.
 
-You can create either resource using the [Azure Portal][create_new_resource] or [Azure CLI][azure_cli].
+You can create either resource using the **Option 1:** [Azure Portal][create_new_resource] or **Option 2:** [Azure CLI][azure_cli].
```

Comment:
Might be nice to format the options on different lines. Here's how it looks now: https://github.com/Azure/azure-sdk-for-java/blob/564cc2cf9cf11318078014aa3d03314039730bcb/sdk/textanalytics/azure-ai-textanalytics/README.md#create-a-text-analytics-resource

Reply:
updated